pkg/abap/aakaas/aakUtils_mock.go

```diff
@@ -0,0 +1,131 @@
+//go:build !release
+// +build !release
```

Comment:
whats the difference between these 2 comments, both prevent inclusion in releases? one enough?

Reply:
Yes, I wrote // +build !release, but then visual studio code just added the other entry. But I guess if VS lets me I can delete the other entry.

Reply:
I tried to just go with '//go:build !release' => jenkins fails again with script return 1 error. And VS does not let me have just '// +build !release' => I would prefer for simplicity just keeping both (otherwise I would need to change it in github directly and change it back each time I would change the file again...)